CONTRIBUTING.md

```diff
@@ -36,6 +36,31 @@ Before submitting a pull request:
 - Check the codebase to ensure that your feature doesn't already exist.
 - Check the pull requests to ensure that another person hasn't already submitted the feature or fix.
 
+## Run Tests
+
+The full test suite requires PHP cli with mongodb extension, a running MongoDB server and a running MySQL server.

```

Comment:
Optional: we can mention that hybrid reference tests will be skipped if no MySQL was found running (added mainly because I can't be bothered to run MySQL 😅)

Reply:
Added